ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/base-fee-input/base-fee-input.js

```diff
@@ -94,10 +94,7 @@ const BaseFeeInput = () => {
   const [editingInGwei, setEditingInGwei] = useState(false);
 
   const [maxBaseFeeGWEI, setMaxBaseFeeGWEI] = useState(() => {
```

Comment:
Condition `estimateUsed !== PRIORITY_LEVELS.CUSTOM` here is needed to ensure that defaults do not override another custom value added by user. Once this is fixed: https://github.com/MetaMask/metamask-extension/pull/13099/files#r770357032 condition in this file will not need any change.